ui/src/app/pages/analysis/notebook-resource-card.tsx

```diff
@@ -42,32 +45,8 @@ export const NotebookResourceCard = fp.flow(
     };
   }
 
-  get resourceType(): ResourceType {
```

Comment:
Lots of common code pushed down to the template.

Many of the actions are very similar across resources, so there's probably an opportunity to reduce the code further.  I'm not planning to do that here.